modules/libsbc/Kconfig

```diff
@@ -4,12 +4,7 @@
 config ZEPHYR_LIBSBC_MODULE
 	bool
 
-config LIBSBC_ENCODER
-	bool "libsbc encoder Support"
+config LIBSBC
+	bool "libsbc Support"
 	help
-	  This option enables the Android SBC encoder library for Bluetooth A2DP
-
-config LIBSBC_DECODER
-	bool "libsbc decoder Support"
-	help
-	  This option enables the Android SBC decoder library for Bluetooth A2DP
+	  This option enables low Complexity Subband Coding (SBC)
```

Comment:
Shouldn't this be "low-complexity subband codec"? (based on https://en.wikipedia.org/wiki/SBC_(codec) )

Reply:
In A2DP spec, The AAC is `Advanced Audio Coding` too. And I ask the question to AI, the answer is:

> Coding refers to the specific algorithm or method used to encode/decode data:
It's the mathematical process and technique
Describes the encoding scheme itself
More academically precise term

> Codec refers to the implementation (software/hardware) that performs coding:
It's the actual encoder/decoder device or program
More of a product or system term
Combines "coder" + "decoder"

So, it seems that the `Codec` is more appropriate as you said. Let me change it.